### PR TITLE
Historical db

### PR DESCRIPTION
**Pull request by kordano**

### CODE REVIEW

**Comment by kordano** [comment]
Should be merged after #39

File: deps.edn [hunk 1]
```diff
@@ -12,24 +12,33 @@
...
+                    :extra-deps  {clj-http/clj-http   {:mvn/version "3.12.3"}
+                                  lambdaisland/kaocha {:mvn/version "1.64.1010"}}
+                    :main-opts   ["-m" "kaocha.runner"]
+                    :exec-fn     kaocha.runner/exec-fn
+                    :exec-args   {}}
+
+           :test {:extra-paths        ["test"]
+                  :extra-deps         {clj-http/clj-http      {:mvn/version "3.12.3"}
+                                       org.clojure/test.check {:mvn/version "1.1.1"}}
+                  lambdaisland/kaocha {:mvn/version "1.64.1010"}}
```
Comment (yflim) [inline deps.edn]: Shouldn't the last line be in `:extra-deps`?

File: src/clj/datahike_server/core.clj [hunk 2]
```diff
@@ -22,4 +22,4 @@
 
   (mount/start)
 
-  (mount/stop))
+  (mount/stop))
```
Comment (yflim) [inline src/clj/datahike_server/core.clj:25]: Spurious formatting change?

File: src/clj/datahike_server/database.clj [hunk 3]
```diff
@@ -39,9 +39,14 @@
 (defn cleanup-databases []
   (stop #'datahike-server.database/conns)
   (doall
```
Comment (yflim) [inline src/clj/datahike_server/database.clj]: Isn't the `doall` obsolete?

**Comment by yflim** [comment]
Looks good apart from the minor points commented on

File: deps.edn (hunk 1 above)
Comment (kordano) [reply deps.edn]: fixed it

File: src/clj/datahike_server/database.clj (hunk 3 above)
Comment (kordano) [reply src/clj/datahike_server/database.clj]: You're right.

File: src/clj/datahike_server/core.clj (hunk 2 above)
Comment (kordano) [reply src/clj/datahike_server/core.clj:25]: yes

File: src/clj/datahike_server/database.clj (hunk 3 above)
Comment (kordano) [reply src/clj/datahike_server/database.clj]: Fixed it.

**Pull request merged**